docs/en/latest/plugins/hmac-auth.md

```diff
@@ -101,24 +105,24 @@ The calculation formula of the signature is `signature = HMAC-SHAx-HEX(secret_ke
 
 > The coding steps of canonical_query_string are as follows:
 
-* Extract the `query` item in the URL, that is, the string "key1 = valve1 & key2 = valve2" after the "?" in the URL.
```

Comment:
Why change this part?

Reply:
All changes except broken links happened due to VSCode auto-format 🤔